docs/user-guide/creating-application/deployment-template.md

```diff
@@ -167,6 +167,13 @@ autoscaling:
 | `enabled` | Set true to enable autoscaling else set false.|
 | `extraMetrics` | Used to give external metrics for autoscaling. |
 
+### Fullname Override
+
+```yaml
+fullnameOverride: app-name
+```
+`fullnameOverride` replaces the name of all the resources that will be created using chart.
```

Comment:
`fullnameOverride` replaces the release fullname created by default by devtron, which is used to construct Kubernetes object names. By default, devtron uses {app-name}-{environment-name} as release fullname.

Reply:
done